style.css

```diff
@@ -0,0 +1,308 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&family=Poppins:wght@300&display=swap');
```

Comment:
# Please Follow the design style;
- The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.
Link to the design 👉 [Original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)